frontend/src/components/place/PlacePreview.vue

```diff
@@ -2,9 +2,17 @@
     <transition name="slide-fade">
         <div class="container place-item" v-if="active">
             <div class="media">
-                <figure class="media-left image is-128x128">
-                    <img v-if="photo" :src="photo">
-                    <img v-else src="../../assets/placeholder_128x128.png">
+                <figure v-if="place.photos !== undefined && place.photos.length" class="media-left image is-128x128">
```

Comment:
better to create `hasPhotos` computed `place.photos !== undefined && place.photos.length`